spec/components/schemas/job.yaml

```diff
@@ -3,13 +3,16 @@ properties:
   object:
     type: string
     enum: ["job"]
-  id:
+  uuid:
     type: string
     format: uuid
   name:
     type: string
     example: Coffee drinker
   color:
     type: string
-    example: null
+    example: '#8CC63E'
     description: HEX representation of color
+  details:
```

Comment:
I don't think we need 'details'. It seems the vast majority of the time there is nothing in that field. I'm unsure if there is a way for users to access that field.

Reply:
WIll remove 'details' as per this morning's convo